packages/vscode/tsup.config.ts

```diff
@@ -1,5 +1,6 @@
 import { copyFileSync, existsSync, mkdirSync, writeFileSync } from 'node:fs'
 import { join } from 'node:path'
+import { env } from 'node:process'
```

Comment:
```suggestion
import process from 'node:process'
```

Let's use `process.env`